### ISSUE TITLE
Map not rendering in Laravel 4.2 . [ SyntaxError: expected property name, got '!' ]

### ISSUE THREAD

**Issue by worlanyodamalie** [bradcornford/Googlmapper#252]
Hello,

Map does not show in Laravel 4.2 , but looking at the console ,this error shows [ SyntaxError: expected property name, got '!' ] . Kindly help me resolve this issue

**Comment by worlanyodamalie** [comment]
Solved this . Realized that in map.blade.php file  for version[v1.25.3] , the blade syntax used was `zoom: {!! $options['zoom'] !!}` instead of `zoom: {{ $options['zoom'] }}` .   I have fixed it locally , but do I need to submit a pull request for it to be resolved. ?    @bradcornford

**Comment by bradcornford** [comment]
Yeah, looks like i might have just cherry-picked a commit from master which may have introduced it.

**Comment by bradcornford** [comment]
I'll get this fixed now.
